tool/data_dir_stat.py

```diff
@@ -0,0 +1,30 @@
+import os
+import datetime
+
+from katikati_pylib.logging import logging
+
+log = None
+
+def get_size_and_upload(CRYPTO_TOKEN_PATH, dirpath, project, firebase_client):
+    global log
+    if log is None:
+        log = logging.Logger(__file__, CRYPTO_TOKEN_PATH)
+    dirname = os.path.basename(dirpath)
+    dir_size = get_dir_size(dirpath)
+    entry = {
+        'project': project,
```

Comment:
can you also add the dirname in the entry as well?

Reply:
Added it